django_rest_passwordreset/views.py

```diff
@@ -160,7 +136,7 @@ def post(self, request, *args, **kwargs):
         if not active_user_found and not getattr(settings, 'DJANGO_REST_PASSWORDRESET_NO_INFORMATION_LEAKAGE', False):
             raise exceptions.ValidationError({
                 'email': [_(
-                    "There is no active user associated with this e-mail address or the password can not be changed")],
+                    "We couldn't find an account associate with that email. Please enter another email account.")],
```

Comment:
Should be `"We couldn't find an account associated with that email. Please try a different e-mail address."`